tests/ui/associated-consts/defaults-cyclic-fail.stderr

```diff
@@ -4,6 +4,11 @@ error[E0391]: cycle detected when const-evaluating + checking `Tr::A`
 LL |     const A: u8 = Self::B;
    |                   ^^^^^^^
    |
+note: see https://rustc-dev-guide.rust-lang.org/overview.html#queries and https://rustc-dev-guide.rust-lang.org/query.html for more information
+  --> $DIR/defaults-cyclic-fail.rs:5:19
+   |
+LL |     const A: u8 = Self::B;
+   |                   ^^^^^^^
```

Comment:
I think we should use an unspanned note, as this span doesn't help for this message